std-bits/table/src/main/java/org/enso/table/aggregations/Concatenate.java

```diff
@@ -0,0 +1,73 @@
+package org.enso.table.aggregations;
+
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.table.Column;
+import org.enso.table.data.table.problems.InvalidAggregation;
+import org.enso.table.data.table.problems.UnquotedDelimiter;
+
+import java.util.List;
+
+public class Concatenate extends AggregateColumn {
+  private final Storage storage;
+  private final String join;
+  private final String prefix;
+  private final String suffix;
+  private final String quote;
+
+  public Concatenate(String name, Column column, String join, String prefix, String suffix, String quote) {
+    super(name, Storage.Type.STRING);
+    this.storage = column.getStorage();
+
+    this.join = join == null ? "" : join;
+    this.prefix = prefix;
+    this.suffix = suffix;
+    this.quote = quote == null ? "" : quote;
+  }
+
+  @Override
+  public Object aggregate(List<Integer> indexes) {
+    StringBuilder current = null;
+    for (int row: indexes) {
+      Object value = storage.getItemBoxed(row);
+      if (value == null || value instanceof String) {
+        String textValue = toQuotedString(value, quote, join);
+
+        if (quote.equals("") && textValue.contains(join)) {
+          this.addProblem(new UnquotedDelimiter(this.getName(), row, "Unquoted delimiter."));
```

Comment:
I think it would be better to have a counter of these problems and only create actual classes at the very end in a 'summarize' step.

Then you could also easily aggregate them - for example instead of 100 `UnquotedDelimiter` instances, you could do `UnquotedDelimiter which_rows` in one instance. Much in the same way as we already do for input problems like out of bound indices where we emit one aggregated problem instead of a series of them.

Plus, probably  could be a bit more performant.